bindings/c/test/mako/mako.h

```diff
@@ -189,4 +189,6 @@ typedef struct {
 /* process type */
 typedef enum { proc_master = 0, proc_worker, proc_stats } proc_type_t;
 
+int run_op_clearrange(FDBTransaction*, char*, char*);
```

Comment:
You could probably just move all of the `run_op_*` functions to the top of mako.c since they only call into FDB library functions.